ynput/ayon-launcher
# Shell script: Fix condition in make.sh script

## Changelog Description
The condition caused `[: -eq: unary operator expected` which should be fixed now.

## Testing notes
1. Make sure `.poetry` is not in root of repository.
2. Run `./tools/make.sh create-env`.
3. Script should not complain about invalid syntax.
